Git-Collabration-Example/ViewController.swift

```diff
@@ -9,9 +9,12 @@ import UIKit
 
 class ViewController: UIViewController {
 
+    @IBOutlet weak var imageView: UIImageView!
+    
     override func viewDidLoad() {
         super.viewDidLoad()
-        // Do any additional setup after loading the view.
+        
```

Comment:
Asking to have this deleted ... just for test collaboration

Reply:
@tonyekas This line has been already removed